test/test_torch_distributed_xla_backend.py

```diff
@@ -155,12 +155,19 @@ def test_broadcast(self):
     opts = dist.BroadcastOptions()
     opts.rootRank = 0
     opts.rootTensor = 0
+
+    # Sync value of `tensor` to remove constants 1 and 0 from graph
+    xm.mark_step()
```

Comment:
Just curios, which part of the above program will set constants 1 and 0 in the graph? Or this is just a safeguard?

Reply:
This makes the HLO more readable in the case of a failure, since it's just taking `tensor` as an input and multiplying it by 1 after this. I also saw some constants in the graph from creating `tensor`, definitely from the `+ 1` but I think also from `arange(2)` because the result is `[0, 1]`. In any case, we don't want to depend on the implementation details of `arange` at all, so I think it's best to sync the graph here.